test-suite/src/test/java/io/micronaut/docs/validation/custom/MyValidatorBean.java

```diff
@@ -28,7 +28,6 @@
 
 // tag::class[]
 @Singleton
-@Introspected
```

Comment:
*cough* import *cough* 😜

Reply:
I swear I'm eventually going to remember to always force IntelliJ to do this. :)

Reply:
[b37b3e3](https://github.com/micronaut-projects/micronaut-validation/pull/336/commits/b37b3e33d30fc08e8f09ec0f5666b8c3facd115f)

Reply:
😡

Reply:
I guess I've just finally found something that Eclipse did better.